docusign-connector-demo/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.axonivy.connector.docusign</groupId>
 	<artifactId>docusign-connector-demo</artifactId>
-	<version>12.0.1-SNAPSHOT</version>
```

Comment:
I think this downgrade commit must be reverted, there is already a 12.0.0 final version published in the field.
If that version comes short of anything, we don't mind to ship a 12.0.1 patch version to address things.

Reply:
Hi @ivy-rew , I'm reverting it now, thank you for your reminding!